openedx/core/djangoapps/kafka_consumer/management/commands/consume_events.py

```diff
@@ -0,0 +1,94 @@
+"""
+Management command for listening to license-manager events and logging them
+"""
+
+import logging
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from confluent_kafka import KafkaError, DeserializingConsumer
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from confluent_kafka.serialization import StringDeserializer
+from openedx.core.djangoapps.kafka_consumer.consumers import getHandler
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Listen for license events from the event bus and log them. Only run on servers where KAFKA_ENABLED is true
+    """
+    help = """
+    This starts a Kafka event consumer that listens to the specified topic and logs all messages it receives. Topic
+    is required.
+
+    example:
+        manage.py ... consume_events -t license-event-prod
+
+    """
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            '-t', '--topic',
+            nargs=1,
+            required=True,
+            help='Topic to consume'
+        )
+
+        parser.add_argument(
+            '-g', '--group_id',
+            nargs=1,
+            required=True,
+            help='Consumer group id'
+        )
+
+    def handle(self, *args, **options):
+        if not settings.KAFKA_ENABLED:
```

Comment:
1. Let's define and document a SettingToggle.
2. We don't need to overthink this if we may iterate, but what is this meant to be a toggle for? Do you want something like `KAFKA_CONSUMERS_ENABLED` as a temporary toggle to turn on/off all Kafka consumer activity?
3. Note: if `KAFKA_CONSUMERS_ENABLED` were on, I think it would be fine to see Kafka configuration errors if Kafka was not set up appropriately.